app/models/fe/concerns/choice_field_concern.rb

```diff
@@ -105,9 +105,8 @@ def validation_class(answer_sheet)
       end
     end
 
-    def display_response(app = nil, joiner = ', ')
+    def display_response(app = nil, _humanize = false)
```

Comment:
@twinge I also forgot to change this method signature.  joiner is irrelevant now since I'm using the responses array directly in conditional_match.  I needed it before because I needed to split the display_responses on ";" rather than "," since choices are somewhat likely to have a comma in them.  Now I just use responses so no need.